File: src/main/java/net/greemdev/core/util/FormatUtil.java
package net.greemdev.core.util;

import static org.bukkit.ChatColor.*;

public class FormatUtil {

    private FormatUtil() {
    }

    public static String getMessagePrefix() {
        return GRAY + "[" + RED + "Core" + GRAY + "]" + RESET + ": ";
    }

    public static String getPermissionError() {
        return getMessagePrefix() + DARK_RED + "You do not have permission to use this command.";
    }
}

File: src/main/kotlin/net/greemdev/core/util/FormatUtil.kt
package net.greemdev.core.util

import org.bukkit.ChatColor.*

public object FormatUtil {
    @JvmStatic public fun getMessagePrefix(): String = "" + GRAY + "[" + RED + "Core" + GRAY + "]" + RESET + ": "
    @JvmStatic public fun getPermissionError(): String = getMessagePrefix() + DARK_RED + "You do not have permission to use this command."
}